contracts/instruments/RibbonThetaVault.sol

```diff
@@ -635,14 +689,22 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
      * @return total balance of the vault, including the amounts locked in third party protocols
      */
     function totalBalance() public view returns (uint256) {
-        return lockedAmount.add(IERC20(asset).balanceOf(address(this)));
+        return lockedAmount.add(assetBalance());
     }
 
     /**
      * @notice Returns the asset balance on the vault. This balance is freely withdrawable by users.
      */
     function assetBalance() public view returns (uint256) {
-        return IERC20(asset).balanceOf(address(this));
+        uint256 balance = IERC20(asset).balanceOf(address(this));
+        if (isYearnWrapped) {
+            balance = balance.add(
+                IERC20(address(collateralToken)).balanceOf(address(this)).mul(
+                    collateralToken.pricePerShare()
```

Comment:
instead of changing everything to yvweth, just change assetBalance to return asset equivalent